kotlin/display-layer-view-state/README.md

```diff
@@ -0,0 +1,48 @@
+# Display layer view state
+
+Determine if a layer is currently being viewed.
+
+![Image of display layer view state](display-layer-view-state.png)
+
+## Use case
+
+The view status includes information on the loading state of layers and whether layers are visible at a given scale. You might change how a layer is displayed in a layer list to communicate whether it is being viewed in the map. For example, you could show a loading spinner next to its name when the view status is LOADING, grey out the name when NOT_VISIBLE or OUT_OF_SCALE, show the name normally when ACTIVE, or with a warning or error icon when the status is WARNING or ERROR.
+
+## How to use the sample
+
+Tap the *Load layer* button to create a new layer and add it to the map. As you pan and zoom around the map, note how the `LayerViewStatus` flags change; for example, `OUT_OF_SCALE` becomes true when the map is scaled outside of the layer's min and max scale range. Tap the *Hide layer* button to hide the layer and observe the view state change to `NOT_VISIBLE`.
```

Comment:
should this include using airplane mode?

Also, airplane mode was the only time I could see a toast message...is there something else I can do to see one? Should that be in the how to use too?

Reply:
Hm, I put the airplane mode stuff in the additional information, do you think it would be better put here?

Reply:
Yeah I think since it's telling the user how to see something we want shown (a layer view state), could be good to include here